WalletWasabi/Helpers/Constants.cs

```diff
@@ -72,7 +72,5 @@ public static BitcoinWitPubKeyAddress GetCoordinatorAddress(Network network)
 
 		public const string ChangeOfSpecialLabelStart = "change of (";
 		public const string ChangeOfSpecialLabelEnd = ")";
-
-		public static Money DustThreshold = Money.Coins(0.0001m);
```

Comment:
I think it is a good thing to have this constant here in order to avoid repeating the `0.0001m` in other places. The only thing I would do is to rename it as `DefaultDustThreshold` instead of `DustThreshold` here and then use it everywhere.

Reply:
What do you mean? If you add a constant here then it's dangerous that someone else would start using it from here and not from the configured one.

Reply:
The code can not prevent developers do something wrong. To prevent such things we have communication and code reviews (we are not a 30 members team). Anyway, it is a suggestion.